addons/point_of_sale/static/src/js/models.js

```diff
@@ -1493,7 +1493,47 @@ exports.PosModel = Backbone.Model.extend({
             'total_included': sign * round_pr(total_included, this.currency.rounding),
         }
     },
+    get_taxes_after_fp: function(taxes_ids){
+        var self = this;
+        var taxes =  this.taxes;
+        var product_taxes = [];
+        _(taxes_ids).each(function(el){
+            var tax = _.detect(taxes, function(t){
+                return t.id === el;
+            });
+            product_taxes.push.apply(product_taxes, self._map_tax_fiscal_position(tax, self.order));
```

Comment:
`self.order` doesn't work in PosModel. Should do `this.get_order()`

Reply:
done